### ISSUE TITLE
@version attribute on iati-activity

### ISSUE THREAD

**Issue by Bjwebb** [IATI/IATI-Schemas#30]
```
        A number indicating the IATI specification version in use.
        Defaults to "1.0" if not specified.  It is required to
        specify this attribute if the document is using features
        specific to an IATI specification other than the initial
        1.0 version.
```

Should this be on both iati-activities and iati-activity. The behaviour if one is defined and the other isn't is not well defined. Moreover, it is not clear if documents with iati-activity elements with different version attributes are permitted, since the documentation says iati-activity@version refers to whether the _document_ uses certain features.

Additionally, the documentation says that it defaults to '1.0'. This version of the schema is not available online.

**Comment by bill-anderson** [comment]
Should be in iati-activities only. (can't see the use case for two activities in the same file employing different versions)

Documentation should say: "defaults to the current version ..."

**Comment by stevieflow** [comment]
Is that a proposal to *remove" @version from iati-activity?  I had understood the rationale for including it was in the use case where people parse out activities (as in the datastore).  In this case, how would we know what version these activity documents relate to (although granted, this is not in use currently)?

**Comment by bill-anderson** [comment]
Yup you're right,  the use case is not for loading into the data store (which can inherit from iati-activities) but in the xml blobs that can be accessed separately.What then is the point of having it in iati-activities??

**Comment by stevieflow** [comment]
Please also note that the description text with this particular attribute is:

"A number indicating the IATI specification version in use. Defaults to "1.0" if not specified. It is required to specify this attribute if the document is using features specific to an IATI specification other than the initial 1.0 version."

Whereas for iati-activities it is just:

"A number indicating the IATI specification version in use."

**Comment by Bjwebb** [comment]
The purpose of having `@version` in `iati-activities` is so that a data user can easily look at a file, and work out what version of the IATI Standard is needed to interpret the file. I don't think it should be possible to have activities of differing versions in a file (as this is problematic for data consumers), but this is not a constraint we currently describe anywhere.

I've written my proposal for this up on the support forum http://support.iatistandard.org/entries/50761977--version-should-be-consistent

**Comment by Bjwebb** [comment]
This has been fixed in 2.01 by requiring both version attributes, and requiring that all versions in a file are the same:
http://support.iatistandard.org/entries/57866638-Tightening-up-on-version
https://github.com/IATI/IATI-Schemas/issues/109
